shared/src/model/exercise/question.js

```diff
@@ -30,6 +31,10 @@ export default class ExerciseQuestion extends BaseModel {
     return this.hasFormat('multiple-choice');
   }
 
+  @computed get isOpenEnded() {
+    return this.hasFormat('open-ended');
```

Comment:
I think we were using free-response (which most questions have because most questions CAN be answered free-response) but to use that you might need logic that says the free-response format is the only format.